tutorials/containers/introducing-microk8s/introducing-microk8s.md

```diff
@@ -0,0 +1,250 @@
+kube-system   service/monitoring-influxdb    ClusterIP   10.152.183.163   <none>        8083/TCP,8086/TCP   49s
+
+NAMESPACE     NAME                                             DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+kube-system   deployment.apps/heapster-v1.5.2                  1         1         1            1           49s
+kube-system   deployment.apps/kube-dns                         1         1         1            1           54s
+kube-system   deployment.apps/kubernetes-dashboard             1         1         1            1           49s
+kube-system   deployment.apps/monitoring-influxdb-grafana-v4   1         1         1            1           49s
+
+NAMESPACE     NAME                                                        DESIRED   CURRENT   READY   AGE
+kube-system   replicaset.apps/heapster-v1.5.2-5995dc5b8c                  1         1         1       29s
+kube-system   replicaset.apps/heapster-v1.5.2-7864d77c47                  0         0         0       37s
+kube-system   replicaset.apps/heapster-v1.5.2-dd4fd4c49                   0         0         0       49s
+kube-system   replicaset.apps/kube-dns-67b548dcff                         1         1         1       54s
+kube-system   replicaset.apps/kubernetes-dashboard-67d4c89764             1         1         1       49s
+kube-system   replicaset.apps/monitoring-influxdb-grafana-v4-8467db6558   1         1         1       49s
+```
+
+### Kubernetes dashboard
+
+As we see above the kubernetes-dashboard service in the kube-system namespace has a ClusterIP of `10.152.183.149` and listens on `443`. The ClusterIP is randomly assigned so if you follow these steps on your host, make sure you check the IP you got. Point your browser to `https://10.152.183.149:443` and you will see the kubernetes dashboard UI.
+
+![Kubernetes Dashboard](images/dashboard.png)
+
+### Grafana dashboard
+
+Let's see an alternative approach in accessing hosted services. The API server proxies our services, here is how to get to them:
+```bash
+> microk8s.kubectl cluster-info
+Kubernetes master is running at http://127.0.0.1:8080
+Heapster is running at http://127.0.0.1:8080/api/v1/namespaces/kube-system/services/heapster/proxy
+KubeDNS is running at http://127.0.0.1:8080/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+Grafana is running at http://127.0.0.1:8080/api/v1/namespaces/kube-system/services/monitoring-grafana/proxy
+InfluxDB is running at http://127.0.0.1:8080/api/v1/namespaces/kube-system/services/monitoring-influxdb:http/proxy
+```
+
+We only need to point our browser to (http://127.0.0.1:8080/api/v1/namespaces/kube-system/services/monitoring-grafana/proxy)
+
+![Grafana Dashboard](images/grafana.png)
+
+
+## Host your first service in Kubernetes
+Duration: 7:00
+
+As we are introducing microk8s we do not use yaml manifests, instead we create a deployment of microbot with two pods via the cli:
```

Comment:
You should explain what a manifest is if you bring it up. And why would we not use one here? For simplicity?

Reply:
I removed the reference to manifests. There is not point in mention them if we are not going to use them.